chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/test/InstanceSupervisor.kt

```diff
@@ -246,8 +243,8 @@ object InstanceActions {
     fun newUser(
         id: String,
         name: String = "No name",
-        avatarUrl: String = "https://gravatar.com/img/2124",
-        customData: CustomData? = null
+        avatarUrl: String = "https://gravatar.com/img/2124"
+//        customData: CustomData? = null
```

Comment:
CustomData should stay, just be moved out from ChatManager (which is gone) to its own file (I have actually done it in v2Skeching branch).

Reply:
I've added it to [it's own file](chatkit-core/src/main/kotlin/com/pusher/chatkit/CustomData.kt) - i must have missed one of the references - have uncommented and fixed :)